tests/general/test.sh

```diff
@@ -64,13 +67,20 @@ rlJournalStart
         else
             rlLogInfo "ANSIBLE_VER not defined - using system ansible if installed"
         fi
-        rolesCloneRepo "$role_path"
+        rlLog "TEST_LOCAL_CHANGES = $TEST_LOCAL_CHANGES"
+        if [ "$TEST_LOCAL_CHANGES" ]; then
```

Comment:
```suggestion
        if [ "$TEST_LOCAL_CHANGES" = true ]; then
```

Reply:
There is an issue with this. I tested setting `TEST_LOCAL_CHANGES: true` env var in a plan, in the test the var has a value of `True` with capital T. So check `= true` doesn't work, and the check `[ "$TEST_LOCAL_CHANGES" ]` works fine.

Reply:
> There is an issue with this. I tested setting `TEST_LOCAL_CHANGES: true` env var in a plan, in the test the var has a value of `True` with capital T. So check `= true` doesn't work, and the check `[ "$TEST_LOCAL_CHANGES" ]` works fine.

Ok - yeah, due to implicit YAML conversion rules, if you want the string value "true" you have to use `TEST_LOCAL_CHANGES: "true"` - the current approach is ok, but if someone does
`TEST_LOCAL_CHANGES: false` or `TEST_LOCAL_CHANGES: "false"`, it will not work as expected.

Reply:
Is this fine?
`if [ "$TEST_LOCAL_CHANGES" == true ] || [ "$TEST_LOCAL_CHANGES" == True ]; then`

Reply:
yes - thanks